validation/main.cpp

```diff
@@ -116,16 +118,35 @@ int main(int argc, char *argv[]) {
             return EXIT_FAILURE;
         }
     }
+
+    QStringList commandList = {"time_settings 0 1 0"};
```

Comment:
You do not want to add this for default, what if someone wants to change the time settings?

Reply:
`gtp-command`s given by the user are added to the end of the list so any other time_settings commands that follow will be done after this one, overwriting what this first one does. Seems to work from when I've used it.
I thought this would be easier than parsing `commandList` after all the user's commands had been added and looking to see if a time_settings one had been provided. It would seem cleaner to not always send this initial command though so I'd be happy to change it if anyone has ideas on a better approach.